### PR TITLE
Introduce sections for the project summary

### PR DESCRIPTION
**Pull request by sven1103**
Provides substantial refactoring for the project summary, increasing readability while also introducing a new UI change.

The project information now displays information in sections with funding information.

### CODE REVIEW

**Review by KochTobi — CHANGES_REQUESTED** [review]
Hi Sven,
I like that you switched to the simple display of information. The code looks good. Sadly it does not work on normal-sized screens as content overflows.

Screenshot 2023-09-27 at 08 04 30

File: vaadinfrontend/frontend/themes/datamanager/components/page-area.css [hunk 1]
```diff
@@ -11,7 +11,7 @@
   max-width: 100%;
   max-height: 100%;
   margin: var(--lumo-space-s);
-  padding: var(--lumo-space-l);
+  padding: var(--lumo-space-xl);
```
[the remark this answers is not in the document]
Comment (sven1103) [reply vaadinfrontend/frontend/themes/datamanager/components/page-area.css:14]: it is within the selector .project, so no affect else where

[the remark this answers is not in the document]
Comment (sven1103) [reply vaadinfrontend/frontend/themes/datamanager/components/page-area.css:14]: Correction: it was moved to the high level page area selector to harmonize the layout.

**Review by KochTobi — APPROVED** [review]
LGTM

**Review by KochTobi — APPROVED** [review]

**Pull request merged**